### ISSUE TITLE
Some question about the definition of batch

### ISSUE THREAD

**Issue by WWXkenmo** [martenlienen/torchode#13]
Thanks for providing such a great tool!
I am not an expert in this field, but I think your work gonna have a great help to the current project I am working on. But I am still curious about if what I am understands are correct.
For example, in one example in your tutorial (https://torchode.readthedocs.io/en/latest/torchdiffeq/), the "batch" means an ODE system with different initial value, so the problem of other solvers (like torchdiffeq) are jointly solve "batches" of ODE system, so would introduce unintended interaction between these Systems, because they are independent (?) but your work could parallel to solve those systems, and even the initial condition, time interval, step size could be different in each system?
If I understand correctly, give a graph (e.g. social network), if I could decoupled them into several independent subsystem (so a subsystem could be regarded as a batch?), and I could use your tool to parallel solve them even with different initial condition and time interval correct?

**Comment by martenlienen** [comment]
Yes, that is correct. If you manage to split your graph into independent subgraphs and batch them, you can choose different initial conditions etc for each of them and each ODE will be solved with its own solver state, e.g. step size and error estimate.

**Comment by WWXkenmo** [comment]
That would be realllllly cool, thanks so much!!
